webapi/src/main/scala/org/knora/webapi/slice/ontology/domain/service/CardinalityService.scala

```diff
@@ -239,19 +228,20 @@ final case class CardinalityServiceLive(
   private def getCardinalityForProperty(classInfo: ClassInfoContentV2, propertyIri: SmartIri): Option[Cardinality] =
     classInfo.directCardinalities.get(propertyIri).map(_.cardinality)
 
-  private def currentCardinalityIfSetIsIncludedInNewCardinality(
+  private def checkCurrentCardinalityIsIncludedInNewCardinalityOrPersistentEntitiesAreCompatible(
```

Comment:
It was already very long one, but now is too long. Maybe instead of too long name add the docstring?

Reply:
I do not understand why a long, descriptive method name should be _too_ long. Would you care to elaborate on your notion that it "now is too long"? What is the threshold?

Reply:
DescriptiveIsOneThingButNowIsReallyHardToReadEspeciallyWeHaveDocstringWhichIsNoBigDealToUse.

Reply:
I guess we have to disagree on whether or not your funny little sentence is hard to read or not, because I do not have a problem with it. I suppose this is my Java heritage paying off, whether this is a good thing or not I am not yet sure.

The problem with the scaladoc I have is not the following:
* It is not readily available when just reading the code, you have to have an IDE open and hover with your mouse over the method before it shows up
* Documenting private methods is sometimes necessary in order to explain what and more often why a method does what it does. However, in this caseI feel that I can express it in the method name.

I suppose we have a different view on what is too long and what is not. Certainly the current name does not violate any of our formatting rules, neither on the definition nor on the call side. Hence, I will keep it for now.

Reply:
We don't have strict rules about how long methods name should be, but I've heard about 50 characters or so. At the end all this is about common sense. IMO more sense here has docstring instead of this what you are trying to force push.